README.md

```diff
@@ -840,17 +840,50 @@ panic: send on closed channel
 
 If you're seeing panics like above, this would be because you're operating on a closed DB. This can happen, if you call `Close()` before sending a write, or multiple times. You should ensure that you only call `Close()` once, and all your read/write operations finish before closing.
 
-- **Are there any Go specific settings that I should use?**
+### Are there any Go specific settings that I should use?
 
-We *highly* recommend setting a high number for GOMAXPROCS, which allows Go to
+We *highly* recommend setting a high number for `GOMAXPROCS`, which allows Go to
 observe the full IOPS throughput provided by modern SSDs. In Dgraph, we have set
 it to 128. For more details, [see this
 thread](https://groups.google.com/d/topic/golang-nuts/jPb_h3TvlKE/discussion).
 
-- **Are there any linux specific settings that I should use?**
-
-We recommend setting max file descriptors to a high number depending upon the expected size of you data.
-
+### Are there any linux specific settings that I should use?
+
+We recommend setting `max file descriptors` to a high number depending upon the expected size of
+your data. On Linux and Mac, you can check the file descriptor limit with `ulimit -n -H` for the
+hard limit and `ulimit -n -S` for the soft limit. A soft limit of `65535` is a good lower bound.
+You can adjust the limit as needed.
+
+### I see "manifest has unsupported version: X (we support Y)" error.
+
+This error means you have a badger directory which was created by an older version of badger and
+you're trying to open in a newer version of badger. The underlying data format can change across
+badger versions and users will have to migrate their data directory.
+Badger data can be migrated from version X of badger to version Y of badger by following the steps
+listed below.
+Assume you were on badger v1.5.5 and you wish to migrate to v2.0.0-rc1 version
```

Comment:
Why `v2.0.0-rc1` and not just `v2.0.0`?
We will release it quite soon anyway

Reply:
Sure. Changed `v2.0.0-rc1` to `v2.0.0`